memory/stressng_mem.py.data/README

```diff
@@ -0,0 +1,51 @@
+ Description:
+-------------
+    - The main purpose of this test aims at stressing the memory component of the target system, using the memory stressors through the stress-ng program.
```

Comment:
wrap this line in 80 char which is a general practice

Reply:
@PraveenPenguin Done